### 自定义异常

```python
class MyException(Exception):
    
    def __init__(self, *args):
        self.args = args
    def __str__(self):
        return repr(self.value)
        
```

